assets/manifest.webmanifest

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "UniquePasswordBuilder",
-    "short_name": "UniquePasswordBuilder",
+    "name": "Unique Password Builder",
+    "short_name": "UPB",
     "description": "A password builder to generate a strong and different password for each website",
-    "start_url": "index.html",
+    "start_url": "index.html?pwa=true",

```

Comment:
I would prefer to use hash so `index.html#pwa` since server is serving the same page.
What do you think ?